Source/Private/prompt.ps1

```diff
@@ -1,6 +1,8 @@
 function prompt {
     # FIRST, make a note if there was an error in the previous command
     [bool]$script:LastSuccess = $?
+	
+    $Script:PowerLineConfig = $(Get-Module PowerLine|Import-Configuration)
```

Comment:
Ha! I was just saying to someone that I needed to do this, because it breaks during debugging.
Looks like that bothered someone else too ;-)